### DISCUSSION TITLE
[Art competition] Bell-LaPadula Model

### DISCUSSION THREAD

**Discussion by DoHunLee1** [prosyslab-classroom/cs348-information-security#418] [Art competition]
![_4726ff30-121e-4588-92f4-9e084f11820a](https://github.com/prosyslab-classroom/cs348-information-security/assets/101397060/fdcd3c44-7be6-4de5-b9e7-6f74ca38d957)

## Short Description
For me, [Bell-LaPadula Model] means [People in computer simulation, but they don't know they are living in simulation ]".

## Long Description (up to 3 sentences)
The subjects in the simulation cannot see the world outside the computer ("No read up"), but the results of the simulation are used to change the world. Also, in the real world, people look at the subjects of simulation, but cannot manipulate them in the middle to obtain accurate simulation results ("No write down").

**Comment by KihongHeo** [comment]
Interesting. I don't clearly understand the "No write down" part. Why can't we obtain the results?
